queenbee/schema/arguments.py

```diff
@@ -293,7 +293,29 @@ class WorkflowArguments(Arguments):
     field for user_data.
     """
     user_data: Dict = Field(
-        None,
+        {},
```

Comment:
`None` is the better option here. Using mutable objects as default argument values can be tricky:

https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
Hmm I see your point. The reasoning behind this was to make the default workflow as complete as possible to avoid having to run endless `if "user_data" in workflow.arguments: etc...` It feels like an ok sacrifice to make the code less verbose and offload the correct `parsing` of objects to Pydantic.